File: baekyeonsu/문자열 내 마음대로 정렬하기.js
function solution(strings, n) {
  strings.sort((a, b) => {
    if (a[n] > b[n]) return 1;
    if (a[n] < b[n]) return -1;
    if (a >= b) return 1;
    if (a < b) return -1;
  });
  return strings;
}

File: kimbyulyi/PGS/Level1/문자열내마음대로정렬하기.java
import java.util.*;

class Solution {
    public static String[] solution(String[] strings, int n) {
        ArrayList<String> arr = new ArrayList<>();

        for (String string : strings) {
	    // 비교할 문자를 각 문자열 맨 앞에 붙여주기
            arr.add("" + string.charAt(n) + string);
        }
        Collections.sort(arr);  //오름차순 정렬

        String[] answer = new String[arr.size()];
        for (int i = 0; i < answer.length; i++) {
            answer[i] = arr.get(i).substring(1);
            System.out.println(answer[i]);
        }
        return answer;
    }
}